src/components/RepositorySearchCombobox.tsx

```diff
@@ -115,6 +115,7 @@ const RepositorySearchCombobox = ({
         <Box>
           <IconButton
             {...getToggleButtonProps()}
+            colorScheme="gray"
```

Comment:
Default color schema is "primary" now, so I'm setting "gray" here to keep the previous color for this element.